core/admin/status-report/maxi-system-status-report.php

```
@@ -1160,8 +1160,8 @@ public function generate_report_text($data) {
 			}
 
 			// Redact full URLs for security in text report
-			$site_url_parts = parse_url(site_url());
-			$home_url_parts = parse_url(home_url());
+			$site_url_parts = wp_parse_url(site_url());
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$site_url_parts'. (undefined)

(UnusedLocalVariable)